src/main/java/io/xpring/xrpl/legacy/javascript/LegacyJavaScriptSigner.java

```diff
@@ -20,9 +20,9 @@
 
     public LegacyJavaScriptSigner() throws JavaScriptLoaderException {
         Context context = JavaScriptLoader.getContext();
+        this.transactionClass = JavaScriptLoader.loadResource("LegacyTransaction", context);
```

Comment:
Was the change in order of loading resources intentional?

Reply:
no. I think i had vague goals of alphabetization but I clearly didn't even nail that. 

Reverted.